tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -94,17 +81,103 @@ def test_ignore_registry(key_string, key_object, tags_to_apply, get_configuratio
+                                                    '"Sending FIM event: ..." event',
+                                    accum_results=2).result()
+
+        assert event[0]['data']['type'] == 'added', f'Wrong event type.'
+        assert event[0]['data']['path'] == os.path.join(root_key, registry, subkey), f'Wrong key path.'
+        assert event[0]['data']['arch'] == '[x32]' if arch == KEY_WOW64_32KEY else '[x64]', f'wrong key arch.'
+
+        assert event[1]['data']['type'] == 'modified', f'Parent key event type not equal'
+        assert event[1]['data']['path'] == os.path.join(root_key, registry), f'Wrong parent key path.'
+        assert event[1]['data']['arch'] == '[x32]' if arch == KEY_WOW64_32KEY else '[x64]', f'Parent key arch not equal.'
+    else:
+        # The ignore event is generated before the event of the parent key.
+        while True: # Look for the ignore event of the created key
+            ignored_key = wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                                                   callback=callback_ignore).result()
+            if ignored_key == os.path.join(key, os.path.join(registry, subkey)):
+                break
+
+        event = wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+                            callback=callback_detect_event,
+                            error_message='Did not receive expected '
+                                            '"Sending FIM event: ..." event',
+                            accum_results=1).result()
+
+        assert event['data']['type'] == 'modified', f'Parent key event type not equal'
+        assert event['data']['path'] == os.path.join(root_key, registry), f'Wrong parent key path.'
+        assert event['data']['arch'] == '[x32]' if arch == KEY_WOW64_32KEY else '[x64]', f'Parent key arch not equal.'
+
+
+@pytest.mark.parametrize('root_key, registry, arch, value, triggers_event, tags_to_apply', [
+    (key, subkey_1, KEY_WOW64_32KEY, "some_value", True, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_64KEY, "some_value", True, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_64KEY, "regex_ignored_value", False, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_32KEY, "regex_ignored_value", False, {'ignore_registry_value'}),
+    (key, subkey_2, KEY_WOW64_64KEY, "regex_ignored_value", False, {'ignore_registry_value'}),
+    (key, subkey_2, KEY_WOW64_64KEY, "some_value", True, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_32KEY, ignore_value, False, {'ignore_registry_value'}),
+    (key, subkey_1, KEY_WOW64_64KEY, ignore_value, False, {'ignore_registry_value'}),
+    (key, subkey_2, KEY_WOW64_64KEY, ignore_value, False, {'ignore_registry_value'})
+])
+def test_ignore_registry_value(root_key, registry, arch, value, triggers_event, tags_to_apply,
+                         get_configuration, configure_environment, restart_syscheckd, wait_for_fim_start):
```

Comment:
This line is under-indented.
```suggestion
                               get_configuration, configure_environment, restart_syscheckd, wait_for_fim_start):
```